app/presenters/idv/by_mail/letter_requested_email_presenter.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Idv
+  module ByMail
+    class LetterRequestedEmailPresenter
+      include ActionView::Helpers::TranslationHelper
+      include Rails.application.routes.url_helpers
+
+      attr_reader :current_user, :service_provider, :url_options
+      def initialize(current_user:, url_options:)
+        @current_user = current_user
+        @url_options = url_options
+        @service_provider = current_user.pending_profile&.initiating_service_provider
+      end
+
+      def sp_name
+        service_provider&.friendly_name
+      end
+
+      def show_sp_contact_instructions?
+        sp_name.present?
+      end
+
+      def show_cta?
+        sign_in_url.present?
+      end
+
+      def sign_in_url
+        if service_provider.present?
+          service_provider_homepage_url
+        else
+          root_url
+        end
+      end
```

Comment:
Author's note: For the rare edge-case where an SP doesn't have a homepage URL configured, we conferred with @artfulaction to still show the call-to-action button